backend/cmd/server.go

```diff
@@ -48,12 +48,25 @@ func main() {
 		oidcClientSecret:      conf.OidcClientSecret,
 		oidcIdpIssuerURL:      conf.OidcIdpIssuerURL,
 		oidcScopes:            strings.Split(conf.OidcScopes, ","),
+		oidcSkipTLSVerify:     conf.OidcSkipTLSVerify,
 		baseURL:               conf.BaseURL,
 		proxyURLs:             strings.Split(conf.ProxyURLs, ","),
 		enableHelm:            conf.EnableHelm,
 		enableDynamicClusters: conf.EnableDynamicClusters,
 		cache:                 cache,
 		kubeConfigStore:       kubeConfigStore,
 		multiplexer:           multiplexer,
-	})
+	}
+
+	if conf.OidcCAFile != "" {
+		caFile, err := os.ReadFile(conf.OidcCAFile)
+		if err != nil {
+			logger.Log(logger.LevelError, nil, err, "reading oidc-ca-file")
+			os.Exit(1)
+		}
+
+		headlampConfig.oidcCACert = string(caFile)
```

Comment:
I wonder if there is a string - []byte issue here? Why not leave it as []byte?

The functions that use it use []byte, and I see at least one case where it's not being cast back to []byte?
> func (s *[CertPool](https://pkg.go.dev/crypto/x509#CertPool)) AppendCertsFromPEM(pemCerts [][byte](https://pkg.go.dev/builtin#byte)) (ok [bool](https://pkg.go.dev/builtin#bool))